run.py

```diff
@@ -17,37 +17,33 @@
 pygame.mixer.init()
 pygame.mixer.music.load(MUSIC_FILE_PATH)
 pygame.mixer.music.set_volume(MUSIC_VOLUME_PERCENTAGE)
-# the -1 argument repeats the song endlessly
-pygame.mixer.music.play(-1)
+pygame.mixer.music.play(-1) # -1 to repeat song endlessly
 
 
 #tank instance
 player = Player(400, 300, 32, 32)
-#append missile
 player_missile = []
 
 #Add a target sprite with random size
 shootingTarget = Target()
 targetSprites.add(shootingTarget)
 
-#timer instance
-# timer_countdown = Timer()
+#set up background object and skip menu background
+game_background = Background(BACKGROUND_IMAGES_FILE_PATHS)
+game_background.increment_level_background()
+
+timer = Timer()
 
 menu = Menu()
 game_state = None
 game_run = True
-while game_run:
-    #background
-    #set up background object
-    background = Background(BACKGROUND_IMAGES_FILE_PATHS)
-    #display the background image underneath everything else
-    display.blit(background.image,background.loc)
 
+while game_run:
     if game_state == START_GAME:
+        if not timer.is_running():
+            timer.start_timer()
 
-        #change to the level background and blit it to screen
-        background.increment_level_background()
-        display.blit(background.image,background.loc)
+        display.blit(game_background.image,game_background.loc)
```

Comment:
Did changing it to this help make the lag issue go away?

Reply:
No, this name change was more because I thought it would help to distinguish between the game background and the menu background.